.scripts/build_steps.sh

```diff
@@ -33,10 +33,11 @@ solver: libmamba
 CONDARC
 export CONDA_LIBMAMBA_SOLVER_NO_CHANNELS_FROM_INSTALLED=1
 
+# Workaround for bug in https://github.com/conda/conda-build/pull/4281
 mamba install --update-specs --yes --quiet --channel conda-forge --strict-channel-priority \
-    pip mamba conda-build conda-forge-ci-setup=4 "conda-build>=24.1"
+    pip mamba 'conda-build<3.23.0' boa conda-forge-ci-setup=3
 mamba update --update-specs --yes --quiet --channel conda-forge --strict-channel-priority \
-    pip mamba conda-build conda-forge-ci-setup=4 "conda-build>=24.1"
+    pip mamba 'conda-build<3.23.0' boa conda-forge-ci-setup=3
```

Comment:
You can persist this (in a way that a rerender will respect) by setting
```yaml
remote_ci_setup:
# Workaround for bug in https://github.com/conda/conda-build/pull/4281
- conda-build <3.23
- conda-forge-ci-setup =3
```
in conda-forge.yml. However, I strongly recommend to fix whatever is necessary to become compatible with newer conda-build, as you're getting very far outside what can be considered supported in any way shape or form (also: you won't be able to use `{{ stdlib("c") }}` with such old conda-build, and this will produce incorrect package metadata, especially once we move to centOS 7 in a month, and remove the sysroot repodata hacks, whence you'd likely start pulling in glibc 2.28 as a baseline)

Reply:
Thanks for your comment, very much appreciated! I will try to investigate what can be done about this workaround we are currently using. I see we are getting very far behind the main schedule (and actually I'm very interested myself in the `{{ stdlib('c') }}` feature so I'd like to be able to use it)

Reply:
Commit https://github.com/conda-forge/root-feedstock/pull/245/commits/b57f4ca261fa2918d0d0304bc7715e85990c94cc removes the need for the old version of conda-build. On my machine I could run the entire feedstock build without failures, hopefully the CI agrees.